script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -120,142 +121,144 @@ No, that's not right.
 -- Page 8, Offset 29005.
 -- ドレスを濡らす朱は<同胞|はらから>どもの心血だけ。
--- TRANSLATION HERE
+The only crimson that may stain her dress is the lifeblood of her brethren.
 }
 [sha:b54f02afdc72543f0e1d4607bb1eabeb937ddb54]{
 -- Page 9, Offset 29006.
 -- 女に許された自由はそれだけ。
--- TRANSLATION HERE
+This is the only freedom allowed to her.
 }
 [sha:3ee4c6acb315d5fce8a564d011e605fa186267c4]{
 -- Page 9, Offset 29007.
 -- 用を済ませ帰還した女は、落ちる星のように城の深部に幽閉される。
--- TRANSLATION HERE
+When she has completed her task, she returns to the castle like a falling star, to be imprisoned in its furthest depths
 }
 [sha:0726bc472df427ca1bc22650745aa97230753c9d]{
 -- Page 9, Offset 29008.
 -- 自身で起きる事さえできない眠りに落ちる。
--- TRANSLATION HERE
+She falls into slumber, unable to even wake up on her own.
 }
 [sha:4edf9b16d96fb6ad911f66ecc7a0237e1c622689]{
 -- Page 9, Offset 29009.
 -- ……白い女はその運命さえ知らず、ただ、遠い目で月を見上げている。
--- TRANSLATION HERE
+Yet the woman in white in unaware of even that fate, merely gazing upon the Moon, her eyes set unto the distance.
 }
 [sha:924823c4f1b09c53931d1c10ea110abf31357898]{
 -- Page 10, Offset 29010.
 -- ―――そこに、永遠を見た、と思った。
--- TRANSLATION HERE
+...There, I think I saw eternity.
 }
 [sha:403ab1445139c70d0b3c52fd68e8056ebbbc7675]{
 -- Page 10, Offset 29011.
 -- 錯覚か。
--- TRANSLATION HERE
+Perhaps I imagined it.
 }
 [sha:77ee0576b2e68d30b8950e48d5ae6f899dc61b7b]{
 -- Page 10, Offset 29012.
 -- おそらくは錯覚だろう。
--- TRANSLATION HERE
+I suspect I must've imagined it.
 }
 [sha:173ac1b94593eecdd74711880ca02b2fcb844389]{
 -- Page 10, Offset 29013.
 -- だがかまわない。幻だろうと真実だろうと同じ事だ。
--- TRANSLATION HERE
+But it concerns me not. Whether reality or illusion, it makes no difference.
 }
 [sha:7a33552e90e6ea9123b015a560432d7eb6d04df6]{
 -- Page 10, Offset 29014.
 -- 網膜に焼きついた像はたとえ死のうと消え去らない。
--- TRANSLATION HERE
+Even were I to die, would the vision seared into my retinas never fade.
 }
 [sha:78071eb4fe7e3a30a9cfa26f0d00a0553f0f8f84]{
 -- Page 10, Offset 29015.
 -- 幽玄の花。この地上でただ一つの、輝ける星の素子。
--- TRANSLATION HERE
+It was a flower of profound grace; the only mote of the shining stars which could be found on Earth.
 }
 [sha:5063e78c261878ae5d57a1f8016a36874d760930]{
 -- Page 11, Offset 29016.
 -- その星を、摘み取ろうと考えた。
--- TRANSLATION HERE
+I thought I ought to pluck this star.
 }
 [sha:b612e4e9fea9320449e9957d64ee3419c40a4dc6]{
 -- Page 11, Offset 29017.
 -- 不滅なる者同士であれば、死が互いを分かつ事はない。
--- TRANSLATION HERE
+Were we both to be imperishable, death would never do us apart.
 }
 [sha:2b4f93ceea8127dfc614c4e016db5298c22c64d4]{
 -- Page 11, Offset 29018.
 -- 血で結ばれた<環|たまき>。
--- TRANSLATION HERE
+A circlet woven from blood.
 }
 [sha:f0a433a6b3aff4b1a4b6be911bfdbe72e6e723af]{
 -- Page 11, Offset 29019.
 -- 殺すにしても殺されるにしても、決して終わる事のない業を結べばよいと。
--- TRANSLATION HERE
+All I had to do was weave myself a destiny that would never end, whether I killed or was killed.
 }
 [sha:e0c96a8d56142902a5488d2963161c1d561b0ddd]{
 -- Page 11, Offset 29020.
 -- ……地獄に落ちるのは当然だ。
--- TRANSLATION HERE
+That I have fallen into hell is most natural.
```

Comment:
"is natural" or "is a natural consequence". 

"is most natural" is kinda awkward